tests/core.py

```diff
@@ -1383,10 +1385,31 @@ def test_send_mime_dryrun(self, mock_smtp, mock_smtp_ssl):
         assert not mock_smtp_ssl.called
 
 
-if 'AIRFLOW_RUNALL_TESTS' in os.environ:
+class HiveServer2Test(unittest.TestCase):
```

Comment:
We may want to run the tests based on what operators are available, so that anyone can run a subset of the suite locally, regardless of if they have Hive set up or not.